sos/report/plugins/openshift_ovn.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2021 Nadia Pinaeva <npinaeva@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class OpenshiftOVN(Plugin, RedHatPlugin):
+    """This plugin is used to collect OCP 4.x OVN logs.
+    """
+    short_desc = 'Openshift OVN'
+    plugin_name = "openshift_ovn"
+    containers = ('ovnkube-master', 'ovn-ipsec')
+    profiles = ('openshift',)
+
+    def setup(self):
+        self.add_copy_spec([
+            "/var/lib/ovn/etc/ovnnb_db.db",
+            "/var/lib/ovn/etc/ovnsb_db.db",
+            "/var/lib/openvswitch/etc/keys",
+            "/var/log/openvswitch/libreswan.log",
+            "/var/log/openvswitch/ovs-monitor-ipsec.log"
+        ])
+
+        if self.container_exists('ovnkube-master'):
+            self.add_cmd_output([
+                self.fmt_container_cmd(
+                    'ovnkube-master',
+                    'ovn-appctl -t /var/run/ovn/ovnnb_db.ctl ' +
+                    'cluster/status OVN_Northbound'),
+                self.fmt_container_cmd(
+                    'ovnkube-master',
+                    'ovn-appctl -t /var/run/ovn/ovnsb_db.ctl ' +
+                    'cluster/status OVN_Southbound')
+            ])
+        if self.container_exists('ovn-ipsec'):
+            self.add_cmd_output([
```

Comment:
Ack, however let me see about adding a `container` parameter to `add_cmd_output()` so this reads a little easier. We could then simply do things like:

```python
self.add_cmd_output([
   'ipsec status',
   'ovs-appctl [...]'
], container='ovn'ipsec')
```

Reply:
Do you want to have this change before this PR is merged, so that I can rebase?

Reply:
Ack. Just merged the PR for this.